providers/shared/components/directory/id.ftl

```diff
@@ -0,0 +1,112 @@
+[#ftl]
+
+[@addComponentDeployment
+    type=DIRECTORY_COMPONENT_TYPE
+    defaultGroup="solution"
+/]
+
+[@addComponent
+    type=DIRECTORY_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type"  : "Description",
+                "Value" : "A managed directory services instance"
+            }
+        ]
+    attributes=
+        [
+            {
+                "Names" : "Engine",
+                "Types" : STRING_TYPE,
+                "Values" : [ "simple" ],
+                "Mandatory" : true
+            },
+            {
+                "Names" : "EnableSSO",
+                "Types" : BOOLEAN_TYPE,
+                "Default" : false
+            },
+            {
+                "Names" : "DsName",
+                "Types" : STRING_TYPE,
+                "Description": "FQDN for the directory service",
+                "Mandatory" : true
+            },
+            {
+                "Names" : "Password",
+                "Types" : STRING_TYPE
+            },
```

Comment:
Is this different to the generated credentials?

Reply:
I don't think it needs to be. Removed